test/tc_selinux_profilebindings_test.go

```diff
@@ -0,0 +1,163 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"fmt"
+	"strings"
+)
+
+func (e *e2e) testCaseSelinuxProfileBinding([]string) {
+	e.selinuxOnlyTestCase()
+	const selinuxTestProfileName = "testpolicy"
+	const selinuxBindingName = "profile-binding-selinux"
+	const testPodName = "selinux-profile-test-pod"
+	selinuxTestProfile := fmt.Sprintf(`
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha2
+kind: SelinuxProfile
+metadata:
+  name: %s
+spec:
+  allow:
+    var_log_t:
+      dir:
+        - open
+        - read
+        - getattr
+        - lock
+        - search
+        - ioctl
+        - add_name
+        - remove_name
+        - write
+      file:
+        - getattr
+        - read
+        - write
+        - append
+        - ioctl
+        - lock
+        - map
+        - open
+        - create
+      sock_file:
+        - getattr
+        - read
+        - write
+        - append
+        - open
+`, selinuxTestProfileName)
+
+	selinuxBinding := fmt.Sprintf(`
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  name: profile-binding-selinux
+spec:
+  profileRef:
+    kind: SelinuxProfile
+    name: %s
+  image: quay.io/wenshen/spo_test:selinux_busybox
+`, selinuxBindingName)
+
+	testPod := fmt.Sprintf(`
+apiVersion: v1
+kind: Pod
+metadata:
+  name: %s
+spec:
+  initContainers:
+  - image: quay.io/wenshen/spo_test:selinux_busybox
+    name: errorLoggerBinding
+    command: ["sh"]
+    args: ["-c", "echo \"Time: $(date). Some error info.\" >> /var/log/test.log || /bin/true"]
+    volumeMounts:
+    - name: varlog
+      mountPath: /var/log
+  containers:
+  - name: pauser
+    image: "gcr.io/google_containers/pause:latest"
+  restartPolicy: Never
+  volumes:
+  - name: varlog
+    hostPath:
+      path: /var/log
+      type: Directory
+`, testPodName)
+
+	e.logf("creating policy")
+	e.writeAndCreate(selinuxTestProfile, "selinuxProfile-test.yml")
+
+	// Let's wait for the policy to be processed
+	e.kubectl("wait", "--timeout", defaultSelinuxOpTimeout,
+		"--for", "condition=ready", "selinuxprofile", selinuxTestProfileName)
+	defer e.kubectl("delete", "selinuxprofile", selinuxTestProfileName)
+
+	e.logf("Creating test profile binding")
+	e.writeAndCreate(selinuxBinding, "selinuxPolicyBinding-test.yml")
+	defer e.kubectl("delete", selinuxBindingName)
+
+	e.logf("Creating test pod")
+	e.writeAndCreate(testPod, "selinuxBindingPod-test.yml")
+	defer e.kubectl("delete", "pod", testPodName)
+
+	e.logf("Waiting for test pod to be initialized")
+	e.waitFor("condition=initialized", "pod", testPodName)
```

Comment:
Why do we separately wait for initialized and then ready? Can't we just wait for condition=ready?

Reply:
> On Wed, Mar 30, 2022 at 12:29:10AM -0700, Sascha Grunert wrote: @jhrozek @JAORMX PTAL
> Yup, I started looking at the PR yesterday, just didn't finish the review. Sorry for the delay.

Thanks for the review!